app/javascript/tests/close_contact/CloseContact.test.js

```diff
@@ -0,0 +1,185 @@
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(1).find('FormLabel').text()).toContain(`Last Name *`);
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(2).find('FormLabel').text()).toContain(`Phone Number`);
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(3).find('FormLabel').text()).toContain(`Email`);
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(4).find('FormLabel').text()).toContain(`Last Date of Exposure`);
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(5).find('FormLabel').text()).toContain(`Assigned User`); // There's also the InfoTooltip
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(5).find('FormLabel').containsMatchingElement(<InfoTooltip />)).toBeTruthy();
+    expect(emptyCCWrapper.find(InfoTooltip).prop('tooltipTextKey')).toEqual('assignedUser');
+
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(6).find('FormLabel').at(0).text()).toContain('Notes');
+    expect(emptyCCWrapper.find(Modal.Body).find('Row').at(6).find('FormLabel').at(1).text()).toContain('2000 characters remaining');
+    expect(emptyCCWrapper.find(Modal.Footer).exists()).toBeTruthy();
+    expect(emptyCCWrapper.find(Modal.Footer).find(Button).at(0).text()).toEqual('Cancel');
+    expect(emptyCCWrapper.find(Modal.Footer).find(Button).at(1).text()).toEqual('Create');
+  })
+
+  it('Properly renders empty fields when an empty close contact is used as a prop', () => {
+    const emptyCCWrapper = getShallowWrapper(mockCloseContact0);
+    const handleChangeSpy = jest.spyOn(emptyCCWrapper.instance(), 'handleChange');
+    const handleDateChangeSpy = jest.spyOn(emptyCCWrapper.instance(), 'handleDateChange');
+
+    expect(emptyCCWrapper.state('showModal')).toBeFalsy();
+    emptyCCWrapper.find(Button).simulate('click');
+    expect(emptyCCWrapper.state('showModal')).toBeTruthy();
+
+    // This iteration assumes the testInputValues order matches the order of the elements in the DOM
+    testInputValues.forEach((iv, index) => {
+      if (iv.inputType === 'DateInput') {
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', iv.value)
+        expect(handleDateChangeSpy).toHaveBeenCalled();
+      } else {
+        emptyCCWrapper.find(Modal.Body).find('Row').at(index).find(iv.inputType).simulate('change', { target: { id: iv.field, value: iv.value } })
+        expect(handleChangeSpy).toHaveBeenCalled();
+      }
+      expect(emptyCCWrapper.state(iv.field)).toEqual(iv.value);
+    })
+  })
+
+  it('Properly creates a the correct assigned user dropdown options', () => {
+    const emptyCCWrapper = getShallowWrapper(mockCloseContact0);
+    emptyCCWrapper.find(Button).simulate('click');
+    ASSIGNED_USERS.forEach((user, userIndex) => {
+      expect(emptyCCWrapper.find(Modal.Body).find('option').at(userIndex).text()).toEqual(`${user}`);
+    })
+  });
+
+  it('Properly clears any values in the modal on close', () => {
```

Comment:
I would break this up and test clearing when adding a close contact and then one for when you edit a close contact